[wasm] AOT tests failing due to an incorrect path for `ILLink.Descriptors.Serialization.xml`

Started failing with [this rolling build](https://dev.azure.com/dnceng-public/public/_build/results?buildId=17272&view=logs&j=55f4943c-f76d-585b-7250-deab324f0a54&t=18a666a9-d033-504f-063f-baac7386b04a).

```
ILLink : error IL1033: XML descriptor file 'D:\a\_work\1\s\eng\testing\ILLinkDescriptors\XmlSerializer\ILLink.Descriptors.Serialization.xml' could not be found'. [D:\a\_work\1\s\src\libraries\System.Private.Xml\tests\System.Private.Xml.Tests.csproj]
##[error]ILLink(0,0): error IL1033: (NETCORE_ENGINEERING_TELEMETRY=Build) XML descriptor file 'D:\a\_work\1\s\eng\testing\ILLinkDescriptors\XmlSerializer\ILLink.Descriptors.Serialization.xml' could not be found'.
D:\a\_work\1\s\.packages\microsoft.net.illink.tasks\7.0.100-1.22451.5\build\Microsoft.NET.ILLink.targets(86,5): error NETSDK1144: Optimizing assemblies for size failed. Optimization can be disabled by setting the PublishTrimmed property to false. [D:\a\_work\1\s\src\libraries\System.Private.Xml\tests\System.Private.Xml.Tests.csproj]
##[error].packages\microsoft.net.illink.tasks\7.0.100-1.22451.5\build\Microsoft.NET.ILLink.targets(86,5): error NETSDK1144: (NETCORE_ENGINEERING_TELEMETRY=Build) Optimizing assemblies for size failed. Optimization can be disabled by setting the PublishTrimmed property to false.
```

This broke in https://github.com/dotnet/runtime/pull/75499 and it didn't get caught by CI because the EAT tests regressed to be running only smoke tests recently. Fix for that issue is https://github.com/dotnet/runtime/pull/75516 .
<!--Known issue error report start -->
### Report
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|0|0|0|
<!--Known issue error report end -->

--- a/src/libraries/System.Private.Xml/tests/System.Private.Xml.Tests.csproj
+++ b/src/libraries/System.Private.Xml/tests/System.Private.Xml.Tests.csproj
@@ -664,10 +664,10 @@
     <ProjectReference Include="$(CommonTestPath)System\Xml\XmlDiff\XmlDiff.csproj" />
     <ProjectReference Include="XmlReaderLib\System.Xml.RW.XmlReaderLib.csproj" />
   </ItemGroup>
-  
+
   <ItemGroup>
     <TrimmerRootAssembly Include="SerializableAssembly" />
-    <TrimmerRootDescriptor Include="$(ILLinkDescriptorsPath)XmlSerializer\ILLink.Descriptors.Serialization.xml" />
+    <TrimmerRootDescriptor Include="$(ILLinkDescriptorsPath)ILLink.Descriptors.Serialization.xml" />
     <TrimmerRootDescriptor Include="ILLink.Descriptors.ModuleCore.xml" />
   </ItemGroup>
 

